WebRoot/js/zimbraAdmin/common/ZaModel.js

```diff
@@ -437,6 +437,133 @@ ZaModel.getMailPollingIntervalChoices = function () {
 
 ZaModel.MAIL_POLLING_INTERVAL_CHOICES = ZaModel.getMailPollingIntervalChoices;
 
+ZaModel.getMailRecallChoices = function () {
+    return [
+        {
+            value: "1",
+            label: "1 " + ZaMsg.LBL_min
+        },
+        {
+            value: "2",
+            label: "2 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "3",
+            label: "3 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "4",
+            label: "4 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "5",
+            label: "5 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "6",
+            label: "6 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "7",
+            label: "7 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "8",
+            label: "8 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "9",
+            label: "9 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "10",
+            label: "10 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "11",
+            label: "11 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "12",
+            label: "12 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "13",
+            label: "13 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "14",
+            label: "14 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "15",
+            label: "15 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "16",
+            label: "16 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "17",
+            label: "17 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "18",
+            label: "18 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "19",
+            label: "19 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "20",
+            label: "20 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "21",
+            label: "21 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "22",
+            label: "22 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "23",
+            label: "23 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "24",
+            label: "24 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "25",
+            label: "25 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "26",
+            label: "26 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "27",
+            label: "27 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "28",
+            label: "28 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "29",
+            label: "29 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "30",
+            label: "30 " + ZaMsg.LBL_minute
```

Comment:
if some language wants to change ordering of words then how it can be done here?
like if some language wants to show minutes 30 instead of 30 minutes

Reply:
I am referring to the polling interval where the time is set and in each language it will show 5 (minutes) as minutes are being used as labels

Reply:
Received input from sakshi. will have looking into this.

Reply:
@silentsakky, I have tried to achieve this by AjxMessageFormat. However I am still confused for Hebrew and Arabic language, both languages are [Bidirectional] written from right to left with the same BiDi “Bidirectional” writing code when it comes to the new translation technologies.
So in order to display the label in dropdown for both the languages it should be "minutes(in arabic) 12" or its okay to display "12 minutes(in arabic)".
<img width="1480" alt="Screenshot 2023-05-18 at 12 22 40 PM" src="https://github.com/Zimbra/zm-admin-console/assets/59682080/96d7b798-488c-4bce-87ca-c36b174c7bae">

Reply:
if you are using placeholders in translation strings then it doesn't matter, as translator can define position of the placeholder

Reply:
created new ticket in order to fix this issue for Arabic and hebrew language i.e. https://jira.corp.synacor.com/browse/ZCS-13449

Reply:
The value of zimbraFeatureMailRecallTime will always be "minute(s)".
Why don't we put "minutes" outside of pull down menu (choices) in the same way as zimbraMailMessageLifetime? (Account > Advanced > Email Retention Policy)
We can use `ZaMsg.LBL_minute`.

![minutes](https://github.com/Zimbra/zm-admin-console/assets/32184593/4d9f84a2-6ee5-409d-a189-7edc59f5b64d)

Reply:
I think `minutes` is okay as unit string even if the value is `1`. The unit of `E-mail message lifetime` is `days` as the screenshot - it does not consider singular or not.